resources/docs/erp_versionsuebergang-source.adoc

```diff
@@ -22,10 +22,11 @@ Folgende Konfigurationen wurden definiert:
 |===
 
 
-IMPORTANT: Dieser Stand ist vorläufig und wird noch ergänzt. Insbesondere der Zeitpunkt der Umstellung von RU-DEV und RU im Übergangszeitraum ist noch zu entscheiden.
-
 image:puml_fd_zustaende_timeline.png[width=100%]
 
+NOTE: Ab 01.07.23 gelten weiterhin die 2022/2023er Profile, jedoch antwortet der Fachdienst nur gemäß 2023er Profil.
+Ab 01.01.24 wird nur noch das 2023er Profil akzeptiert.
```

Comment:
Ich bin mir unschlüssig, ob wir diese Note hier drin haben sollten. 
Diese Information sollte man ja aus der Tabelle ablesen. Da kämen ja dann fragen welcher Fachdienst (RU/PU?). 
Unter "wichtige Bemerkungen" haben wir ja auch den Hinweis "Ab Konfiguration "B" antwortet der Fachdienst immer mit den neuen Profilversionen von Task, AuditEvent, ChargeItems, Consent, auch wenn diese auf alte Profile verweisen"

Reply:
Nimmst du die Note noch raus oder willst du es trotzdem drin lassen?

Reply:
Die Note ist zu unspezifisch. Wie wir in unserer Tabelle sehen, ist jede Ressource einzeln zu betrachten.

Reply:
Ich nehme sie gerne heraus